.gitignore

```diff
@@ -1,12 +1,13 @@
 /build
+intra/split/example/example
 
 # General
 .DS_Store
 .AppleDouble
 .LSOverride
 
 # Icon must end with two \r
-Icon

+Icon
```

Comment:
Is the comment accurate?

Reply:
Hm, I don't know what this is about.  It looks like my editor did this by itself.  Only vim seems to preserve this weird line ending.  (Fixed.)